packages/strapi-plugin-content-type-builder/admin/src/containers/RelationFormGroup/index.js

```diff
@@ -0,0 +1,393 @@
+/**
+ *
+ * RelationFormGroup
+ *
+ */
+
+import React from 'react';
```

Comment:
This component seems very similar (at the code level not sure about the ui) so if the ui doesn't differ much from the relations modal don't you think it would be great to make the relation form smarter so it can handle both. It's not a hurry but I think it could be great in an update at some point.

Reply:
You're right. They are very similar but some little things are different for now so I note to change it asap for it to handle both.